GETTING_STARTED.md

```diff
@@ -1,109 +1,86 @@
 
 ```python
-  # Authentication email and password
-  client = C8Client(protocol='https', host='gdn.macrometa.io', port=443,
-   email="user@example.com", password="XXXXX")
+from c8 import C8Client
+client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443, apikey=<your api key>)
 ```
+## Create a collection
+A [Collection](https://macrometa.com/docs/collections/) consists of documents, and they can be local(only stores data in one region) or global(replicates data and maintains consistency across regions). Macrometa offers several types of collections such as [Document Store](https://macrometa.com/docs/collections/documents/), [Key-Value Store](https://macrometa.com/docs/collections/keyvalue/), [Dynamo Table](https://macrometa.com/docs/collections/dynamo/create-dynamo-table) and [Graph-Edge collection](https://macrometa.com/docs/collections/graphs/). For this example we are going to create a [Document Store](https://macrometa.com/docs/collections/documents/).
 
-2. Using bearer token:
+```python
+client.create_collection(name='employees', sync=False, edge=False, local_collection=False, stream=False)
+```
+# CRUD Operations for collections
+## Insert Document
+Documents are stored in collections. Macrometa GDN is schemaless, so you do not need to define valid document attributes when creating your collection. Documents with completely different structures can be stored in the same collection.
 
+**Inserting docs in 'employees' collection:**
 ```python
-# Authentication with bearer token
-client = C8Client(protocol='https', host='gdn.macrometa.io', port=443,
- token=<your token>)
+docs = [
+    {'_key': 'Han', 'firstname': 'Han', 'lastname': 'Solo', 'email': 'han.solo@macrfabricometa.io', 'age': 35,
+     'role': 'Manager'},
+    {'_key': 'Bruce', 'firstname': 'Bruce', 'lastname': 'Wayne', 'email': 'bruce.wayne@mfabricacrometa.io', 'age': 40,
+     'role': 'Developer', 'phone': '1-999-888-9999'},
+    {'_key': 'Jon', 'firstname': 'Jon', 'lastname': 'Doe', 'email': 'jon.doe@macrfabricometa.io', 'age': 25,
+     'role': 'Manager'},
+    {'_key': 'Zoe', 'firstname': 'Zoe', 'lastname': 'Hazim', 'email': 'zoe.hazim@macrfabricometa.io', 'age': 20,
+     'role': 'Director'},
+    {'_key': 'Emma', 'firstname': 'Emma', 'lastname': 'Watson', 'email': 'emma.watson@macrfabricometa.io', 'age': 25,
+     'role': 'Director'}
+]
+client.insert_document(collection_name='employees', document=docs)
 ```
 
-3. Using API key:
+## Read Document
+Macrometa is robust. It uses [C8QL](https://macrometa.com/docs/queryworkers/c8ql/) to create, retrieve, modify and delete data stored in the globally distributed platform. This example demonstrates how to retrieve a particular document.
```

Comment:
I don't think we should refer to c8ql  in any of the document/collection api calls.. c8ql should be referred to in the query sections I guess...  We can add many other useful references around the document section (where ever you feel the right place to be):
https://macrometa.com/docs/collections/documents/
https://macrometa.com/docs/collections/documents/add-document
https://macrometa.com/docs/collections/documents/document-store-data